app/controllers/AppInit.go

```diff
@@ -1,26 +1,26 @@
 package controllers
 
 import (
-	"database/sql"
+	"github.com/gocql/gocql"
 	_"github.com/go-sql-driver/mysql"
-	"fmt"
 	"github.com/revel/revel"
+	"Revel-AddressBook/app/models"
+	"fmt"
 )
 type AppInit struct {
 	*revel.Controller
 }
-var Database *sql.DB
-func InitDB() {
+func InitDB(){
```

Comment:
Is it possible to have InitDB in models package? If yes please make SessionVariable a private variable. I think its name should be DbSession or so.